src/pages/firefox-reality/components/_download.js

```diff
@@ -95,21 +97,38 @@ const Download = () => (
       img={htc}
       text="Get it for Viveport"
       alt="Viveport"
+      deviceType="Standalone VR"
       modalText="Please search for 'Firefox Reality' in the Viveport store"
     />
     <DownloadItem
       img={oculus}
       text="Get it for Oculus Go"
       alt="Oculus"
+      deviceType="Standalone VR"
       link="https://www.oculus.com/experiences/go/2208418715853974/"
     />
     <DownloadItem
       isModal
       img={pico}
       text="Get it for Pico"
       alt="Pico"
+      deviceType="Standalone VR"
       modalText="Firefox Reality is coming soon to the Pico store"
     />
+    <DownloadItem
+      img={htc}
+      text="Get it for Viveport"
+      alt="Viveport PC"
+      deviceType="Tethered VR"
+      link="about:blank"
```

Comment:
Did you want to put in the final link

Reply:
lol, in fact, i did want to do that :-) (forgot to save the file)